### DISCUSSION TITLE
ajaxminicart

### DISCUSSION THREAD

**Discussion by Danieleeffe1** [OpenMage/magento-lts#3668] [Extensions & Modules]
I found this ajax cart for magento 1.9

site download
https://www.magecloud.net/marketplace/extension/ajax-mini-cart-ajax-cart/

file
[Datta_Ajaxminicart-1.0.0.zip](https://github.com/OpenMage/magento-lts/files/13412669/Datta_Ajaxminicart-1.0.0.zip)

I installed it in openmage and it seems to work fine
being a very old plug-in it is not optimized for mobile devices.

the plugin has a serious usability problem

the problem:
the minicart is only displayed if you hover over it with the mouse, this is fine for those who use a desktop device but not for mobile where it is necessary to use the "onclick" function.

I tried to modify this function but the code calls jquery functions, I can concentrate on the css, but as for the rest I have absolutely no skills.

Does anyone want to solve this problem with me to make a modified version of this free plug-in available?

**Comment by addison74** [comment]
Thank for sharing.

How did you get the extension? Did you create an account then you were able to download everything that was free?

**Reply by Danieleeffe1** [reply]
yes

**Reply by Danieleeffe1** [reply]
i found the github repository 

https://github.com/Datta/ajax-mini-cart

I noticed these problems:

1) in the Magento "new products" widget the "add to cart" button is visible for all products (even configurable ones)

Under normal conditions the system sends you back to the product page to configure the product

With this mini-kart active the system generates an error.

2) js libraries create conflicts with standard libraries
